include/mbedtls/gcm.h

```diff
@@ -217,4 +219,8 @@ int mbedtls_gcm_self_test( int verbose );
 }
 #endif
 
+#else  /* !MBEDTLS_GCM_ALT */
+#include "gcm_alt.h"
+#endif /* !MBEDTLS_GCM_ALT */
```

Comment:
I do not think that we should include the function `int mbedtls_gcm_self_test( int verbose )` inside `MBEDTLS_GCM_ALT` as this is simply a test and porters will be just copying it as it is in the .c file.

Reply:
Yup, good catch. I made sure to include it in `gcm.c` (whether or not `MBEDTLS_GCM_ALT` is used), but didn't declare the prototype in `gcm.h` if `MBEDTLS_GCM_ALT` was defined.